src/main.lua

```diff
@@ -540,6 +548,7 @@ addEventingHandler(ActionMap.Transfer, utils.hasMatchingTag("Action", ActionMap.
 			Action = "Debit-Notice",
 			Recipient = recipient,
 			Quantity = msg.Tags.Quantity,
+			["Allow-Unsafe-Addresses"] = tostring(allowUnsafeAddresses),
```

Comment:
Is this logically significant?

Reply:
[e7e4e0a](https://github.com/ar-io/ar-io-network-process/pull/216/commits/e7e4e0a6e5d0305401b706cd5d3b8be13764628e)